external_config/slack.md

```diff
@@ -190,3 +190,10 @@ This should be added as a **global** shortcut.
 - **Request URL:** base URL + /slack-command
 - **Short Description:** Broadcast helpline status to channels or volunteers
 - **Usage Hint:** channel-status|volunteer-status [optional preamble message]
+
+#### Follow-up
+
+- **Command:** /follow-up
```

Comment:
I didn't want to do `/needs-follow-up` because the you couldn't to `/n` enter enter and take advantage of auto-complete.  This way you can to `/fo` enter enter.  I couldn't think a single-letter command name that took less than 2 chars..

Reply:
sounds good thanks for explaining!